src/client/src/ui/spotTile/components/PriceButton.tsx

```diff
@@ -1,27 +1,57 @@
 import React from 'react'
-import { styled } from 'rt-theme'
+import { styled, Theme } from 'rt-theme'
 import { Direction } from 'rt-types'
+import { keyframes, css } from 'styled-components'
+import { RfqState } from './types'
 
 const hoverColors = {
   [Direction.Buy]: 'blue',
   [Direction.Sell]: 'red',
 }
 
-export const TradeButton = styled.button<{ direction: Direction }>`
+const backgroundEffectKeyframes = ({
```

Comment:
Suggestion: all styled components could be move to its own file for easier reading.

Reply:
Done